x-pack/plugins/security_solution/public/common/components/event_details/cti_details/helpers.tsx

```diff
@@ -68,14 +65,15 @@ export const getEnrichmentValue = (enrichment: CtiEnrichment, field: string) =>
   getFirstElement(enrichment[field]) as string | undefined;
 
 /**
- * These fields (e.g. 'x') may be in one of two keys depending on whether it's
- * a new enrichment ('threatintel.indicator.x') or an old indicator alert
- * (simply 'x'). Once enrichment has been normalized and we support the new ECS
- * fields, this value should always be 'indicator.x';
+ * These fields (e.g. 'indicator.ip') may be in one of three places depending on whether it's:
+ *   * a queried, legacy filebeat indicator ('threatintel.indicator.ip')
+ *   * a queried, ECS 1.11 filebeat indicator ('threat.indicator.ip')
+ *   * an existing indicator from an enriched alert ('indicator.ip')
  */
 export const getShimmedIndicatorValue = (enrichment: CtiEnrichment, field: string) =>
   getEnrichmentValue(enrichment, field) ||
-  getEnrichmentValue(enrichment, `${DEFAULT_INDICATOR_SOURCE_PATH}.${field}`);
+  getEnrichmentValue(enrichment, `threatintel.${field}`) ||
```

Comment:
That's a good point; I had chosen to check the legacy one first because none of our data shippers actually generate `threat.indicator` yet. If/when that happens, the following also need to change:

1. Prebuilt indicator rule logic
2. default indicator rule `threat_indicator_path`